base/src/expressions/parser/mod.rs

```diff
@@ -111,7 +112,10 @@ pub enum Node {
     },
     RangeKind {
         sheet_name: Option<String>,
+        // index into vector (0-based)
         sheet_index: u32,
+        // coordinates of opposite corners of the range, e.g. A1:C4.
+        // true if cell 1 is anchored
```

Comment:
`true` here means that the reference is something like `C$4` notice the `$` sign, making the reference 'absolute'